### PR TITLE
Issue 4775 -plugin entryuuid failing

### PR DESCRIPTION
**Pull request by kimettog**
Issue 4775 -plugin entryuuid failing

Bug Description:

Bz 2047171 - dsconf tool should suppport the option to execute "plugin entryuuid  fixup".

Fix Description:

Add missing entryuuid cli #4775

relates: https://github.com/389ds/389-ds-base/issues/4775

Author: Gilbert Kimetto

Reviewed by: droideck

### CODE REVIEW

**Comment by Firstyear** [comment]
I'm assuming that this test at the moment fails in some manner? I thought that we did support the fixup?

File: dirsrvtests/tests/suites/entryuuid/basic_test.py [hunk 1]
```diff
@@ -32,6 +36,37 @@
 UUID_MIN = "00000000-0000-0000-0000-000000000000"
 UUID_MAX = "ffffffff-ffff-ffff-ffff-ffffffffffff"
 
+@pytest.mark.skipif(ds_is_older('1.4.3.27'), reason="Entryuuid is not available in older versions")
+def test_entryuuid_plugin(topology):
```
Comment (droideck) [inline dirsrvtests/tests/suites/entryuuid/basic_test.py]: I would mention in the test name that the test is about CLI as we already have EntryUUID fixup test here.

**Review by droideck — APPROVED** [review]
The rest looks good!

**Comment by kimettog** [comment]
> I'm assuming that this test at the moment fails in some manner? I thought that we did support the fixup?

Hi @Firstyear we are adding a new test here but I will change the name of the test to be more clear as @droideck  recomemended. Thank you

File: dirsrvtests/tests/suites/entryuuid/basic_test.py (hunk 1 above)
Comment (kimettog) [reply dirsrvtests/tests/suites/entryuuid/basic_test.py]: Thank you @droideck ! I updated the test name to reflect its the CLI

**Comment by Firstyear** [comment]
Yeah, no issues from me.

**Review by Firstyear — APPROVED** [review]

**Comment by kimettog** [comment]
> Yeah, no issues from me.

Thank you @Firstyear

**Pull request merged**